style/icons/shield40_us_sc.svg

```diff
@@ -1,5 +1,7 @@
- <path d="m1.5688 2.7279h21.982a0.7105 0.7105 0 0 1 0.7105 0.7105v14.967a0.70938 0.70938 0 0 1-0.70938 0.70938h-21.982a0.7105 0.7105 0 0 1-0.7105-0.7105v-14.967a0.70938 0.70938 0 0 1 0.70938-0.70938z" fill="#fff"/>
+ <rect style="fill:#fff;fill-rule:evenodd;stroke:none" width="23" height="16" x="1" y="3" rx="1" ry="1"/>
+ <path d="m 10.636218,1.0000356 c -0.245265,0.1437346 -0.528785,0.2161394 -0.8105535,0.2851691 -0.151522,0.1551279 -0.544236,0.3422979 -0.351574,0.6074659 0.178332,0.065727 0.189795,0.098715 0.253918,0.1816521 0.06807,0.088033 0.211085,0.113297 0.2910445,0.1562605 0.137378,0.2945947 0.226095,0.6434681 0.488308,0.8535822 0.273942,0.1369891 0.303619,0.4588807 0.570337,0.5820639 0.165539,0.1661812 0.112168,0.3841007 0.30472,0.5137095 0.110993,0.2206522 0.427929,0.2548921 0.419954,0.552762 0.118945,0.171564 -0.01283,0.4790571 0.279319,0.5488617 0.226225,0.1864551 0.0015,0.6029405 0.189464,0.7187983 0.04771,-0.1092802 0.06216,-0.1427147 0.06446,-0.1504002 0.24233,0.2415987 0.652772,-0.3861272 0.668024,-0.6211353 0.427676,0.1263894 0.754417,-0.3416584 0.73834,-0.4570607 0.24583,0.035617 0.649575,-0.3767915 0.929762,-0.5859759 0.123432,-0.1567755 -0.06085,-0.1430521 -0.0586,-0.1718865 C 14.81795,3.5885423 15.251696,3.3346847 15.540927,3.103669 15.085596,2.6006603 14.589963,2.1350291 14.10918,1.656297 13.629898,1.6532841 13.14694,1.6335796 12.667683,1.6309054 12.781401,1.3823012 12.48776,1.1382473 12.390313,1.2090021 12.358171,1.1049304 12.416693,1.0630548 12.251632,1.050789 L 11.70277,1.044926 10.636298,1 Z" fill="#003478" style="fill:none;stroke:#fff;stroke-width:1;stroke-miterlimit:4;stroke-dasharray:none"/>
+ <path d="m 10.636218,1.0000356 c -0.245265,0.1437346 -0.528785,0.2161394 -0.8105535,0.2851691 -0.151522,0.1551279 -0.544236,0.3422979 -0.351574,0.6074659 0.178332,0.065727 0.189795,0.098715 0.253918,0.1816521 0.06807,0.088033 0.211085,0.113297 0.2910445,0.1562605 0.137378,0.2945947 0.226095,0.6434681 0.488308,0.8535822 0.273942,0.1369891 0.303619,0.4588807 0.570337,0.5820639 0.165539,0.1661812 0.112168,0.3841007 0.30472,0.5137095 0.110993,0.2206522 0.427929,0.2548921 0.419954,0.552762 0.118945,0.171564 -0.01283,0.4790571 0.279319,0.5488617 0.226225,0.1864551 0.0015,0.6029405 0.189464,0.7187983 0.04771,-0.1092802 0.06216,-0.1427147 0.06446,-0.1504002 0.24233,0.2415987 0.652772,-0.3861272 0.668024,-0.6211353 0.427676,0.1263894 0.754417,-0.3416584 0.73834,-0.4570607 0.24583,0.035617 0.649575,-0.3767915 0.929762,-0.5859759 0.123432,-0.1567755 -0.06085,-0.1430521 -0.0586,-0.1718865 C 14.81795,3.5885423 15.251696,3.3346847 15.540927,3.103669 15.085596,2.6006603 14.589963,2.1350291 14.10918,1.656297 13.629898,1.6532841 13.14694,1.6335796 12.667683,1.6309054 12.781401,1.3823012 12.48776,1.1382473 12.390313,1.2090021 12.358171,1.1049304 12.416693,1.0630548 12.251632,1.050789 L 11.70277,1.044926 10.636298,1 Z" fill="#003478" style="fill:#003f87;stroke:none;stroke-width:1;stroke-miterlimit:4;stroke-dasharray:none"/>
```

Comment:
In Rebusurance, I simplified the South Carolina state shape to be more angular. It isn’t strictly necessary to simplify the shape, but it made the shield look bolder at that very small size.

Reply:
More angular South Carolina:
![Screenshot from 2022-05-11 21-37-39](https://user-images.githubusercontent.com/1732117/167974648-f06ea875-4578-48d4-b7de-5cbe8a758efe.png) ![Screenshot from 2022-05-11 21-37-54](https://user-images.githubusercontent.com/1732117/167974651-c6ec1c9b-297c-4774-8ecb-01a5cfd7c0d9.png)